Description: Disable surfaces when KMS is used
 This is not working properly at the moment.
Author: Dave Airlie <airlied@redhat.com>
Origin: vendor, http://pkgs.fedoraproject.org/cgit/rpms/xorg-x11-drv-qxl.git/tree/no-surfaces-kms.patch
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=801081

--- a/src/qxl_uxa.c
+++ b/src/qxl_uxa.c
@@ -414,6 +414,8 @@ qxl_create_pixmap (ScreenPtr screen, int
     ErrorF ("Create pixmap: %d %d @ %d (usage: %d)\n", w, h, depth, usage);
 #endif
 
+    if (qxl->kms_enabled)
+	goto fallback;
     if (uxa_swapped_out (screen))
 	goto fallback;
 
